application/src/main/java/org/thingsboard/server/service/subscription/TbSubscriptionUtils.java

```diff
@@ -82,10 +82,12 @@ public static ToCoreMsg toSubEventProto(String serviceId, TbEntitySubEvent event
         return ToCoreMsg.newBuilder().setToSubscriptionMgrMsg(msgBuilder).build();
     }
 
-    public static ToCoreNotificationMsg toProto(UUID id, int seqNumber, TbEntityUpdatesInfo update) {
+    public static ToCoreNotificationMsg toProto(TenantId tenantId, UUID id, int seqNumber, TbEntityUpdatesInfo update) {
```

Comment:
Remove all code that is not related to the scope of changes. Create new PR and cover this with TESTS!!!